go/src/koding/remoteapi/models/j_group_role.go

```diff
@@ -16,7 +16,7 @@ type JGroupRole struct {
 	// id
 	ID string `json:"_id,omitempty"`
 
-	// is configurable
+	// is configureable
```

Comment:
Since it's a field of jRole I'm not sure I can't just fix it and be backward compatible.

```
workers/social/lib/social/models/group/role.coffee
16:      isConfigureable :
26:    { title : 'moderator',  isDefault : yes,  isConfigureable: yes }
```

@gokmen wdyt?